distribution/docs/src/main/resources/_contents/_plugins/resource-uri-policy-plugin-contents.adoc

```diff
@@ -0,0 +1,18 @@
+==== Resource URI Policy Plugin
+
+The Resource URI Policy Plugin configures the attributes required for users to perform Create, Read, Update, and Delete operations on the catalog.
+
```

Comment:
Is this true? Or, does it configure the attributes required to update a metacard's resource uri or create a metacard with a resource uri?